Algebraic Topology/algebraic-topology.tex

```diff
@@ -224,18 +225,18 @@ \section{Homotopy}
 \end{thm}
 \begin{thm}
   If $X$ is path connected, then for any $x_a, x_b \in X$,
-  $$\Pi_1(X, x_a) \isom \Pi_1(X, x_b).$$
-  notation If $X$ is path connected, we can denote the fundamental
+  $$\Pi_1(X, x_a) \isom \Pi_1(X, x_b).$$ %
+  Notation: If $X$ is path connected, we can denote the fundamental
   group with $\pi_1(X)$ or $\pi_1 X$.
 \end{thm}
 \begin{example}
-  pretty picture
+  pretty picture %
 
   This isomorphism is not canonical
   
 
   Note that $\pi_1 X$ is only well defined up to inner automorphism.
-  ((Chris, what does this mean?))
+  ((Chris, what does this mean?)) %
```

Comment:
So, I am not confident here, but I believe this refers to the fact that if you have some f \in \pi_1(X,x_0) and some path p from x_0 to x_1, then p*f*p^{-1} \in \pi_1(X,x_1).